workloads/vars/http.yml

```diff
@@ -0,0 +1,37 @@
+---
+###############################################################################
+# Ansible SSH variables.
+###############################################################################
+ansible_public_key_file: "{{ lookup('env', 'PUBLIC_KEY')|default('~/.ssh/id_rsa.pub', true) }}"
+ansible_private_key_file: "{{ lookup('env', 'PRIVATE_KEY')|default('~/.ssh/id_rsa', true) }}"
+
+orchestration_user: "{{ lookup('env', 'ORCHESTRATION_USER')|default('root', true) }}"
+###############################################################################
+# RHCOS http workload variables.
+###############################################################################
+
+# Container image in use
+workload_image: "{{ lookup('env', 'WORKLOAD_IMAGE')|default('quay.io/openshift-scale/scale-ci-workload', true) }}"
```

Comment:
Does this need to be configurable for every workload?  Also, the variable name seems to be slightly misleading.  `workload_job_node_selector` is a boolean (not an actual node selector) that makes the workload job to be placed on nodes labelled `node-role.kubernetes.io/pbench`.

Reply:
Yeah it is not the best name for a variable, mostly because I wanted to be able to achieve what you mentioned however at the time I didn't invest further into making the node selector completely configurable.  At the time a boolean enabling/disabling the node selector for our pbench/workload node was easiest.  If you'd like to add that functionality, lets get another pr.